app/views/associate_ucr.scala.html

```diff
@@ -26,6 +26,7 @@
         govukButton: GovukButton,
         govukRadios: GovukRadios,
         govukInput: GovukInput,
+        exportsInputText: components.gds.exportsInputText,
         errorSummary: components.errorSummary,
         formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF)
```

Comment:
What do you think about importing those elements instead of putting whole path here?

Reply:
```
@import components.gds._
@import components._
```
?

Reply:
Yup, and `uk.gov.hmrc.play.views.html.helpers._`

Or specific elements like `@import components.errorSummary`, but not sure if in this case there is no warning

Reply:
Done.